### ISSUE TITLE
Mekanism power sources?

### ISSUE THREAD

**Issue by squarism** [MightyPirates/OpenComputers#903]
I could be doing something really stupid but I've tried lots of different permutations.  Placing a full energy cube doesn't power the computer?  I've tried different cables and the power converter.

![2015-02-08_16 16 59](https://cloud.githubusercontent.com/assets/64467/6099480/3e38c624-afae-11e4-9fa3-23e2286f2027.png)

Clicking power gives the error in the log window `Not enough power`.  This isn't a mod pack but I have the following loaded:
- OC 1.7.10-1.4.8.33-universal
- Mekanism-1.7.10-7.1.1.127
- EnderIO 1.7.10-2.2.6.317

I will try another mod that provides power.  I've already found that Ender IO can't be used by itself as a power mod for OpenComputers.  I had tried ThermalExpansion but the installation is sort of tricky.  So I was hoping to just use Mekanism.

Please tell me if this layout is or isn't supposed to work.  That would really help out.  :cake:

**Comment by helpiminabox** [comment]
Judging just by that image, the energy cube is facing the wrong way. It only outputs from one side.

**Comment by squarism** [comment]
I thought that too, so I've flipped it with the same result.

![2015-02-08_16 41 04](https://cloud.githubusercontent.com/assets/64467/6099598/513a3066-afb1-11e4-890e-383926a43745.png)

**Comment by squarism** [comment]
At first, I was indeed using the wrong side.  That's my fault, he mentioned that in the 1.3 tutorial video.  Unfortunately thought, flipping it every which way doesn't do anything.

![2015-02-08_16 43 47](https://cloud.githubusercontent.com/assets/64467/6099617/ac76a888-afb1-11e4-9f2f-01c02ba598df.png)

I have since done a clean instance with MultiMC using just the mods listed and the problem is still happening.

**Comment by squarism** [comment]
I downloaded a mod pack with ATLauncher just to see a mod stack work.  It does.  So we can probably rule out user error.  :)  I basically recreated the image above, was able to boot the OS, load from floppy and it stayed up.  The first time I've gotten the power button to work!  What an awesome mod!

Here are the compatible versions from the mod pack.

```
OpenComputers-MC1.7.10-1.4.1.14-universal.jar
Mekanism-1.7.10-7.1.1.133.jar
MekanismGenerators-1.7.10-7.1.1.133.jar
MekanismTools-1.7.10-7.1.1.133.jar`
```

The cube didn't care which way it faced btw.  Next, I'll try rolling back my multimc version to these versions I listed here.

**Comment by gjgfuj** [comment]
Also Ender IO is able to be used as a power mod for OC, so I dunno what
you're saying.

**Comment by squarism** [comment]
Well, I tried enderio's power options and they didn't work.  [This thread](http://www.curse.com/mc-mods/minecraft/223008-opencomputers?page=2) said the same:

> Question: Can it [opencomputers] work with Ender IO power?
> 
> Answer: Uhm, as in when EnderIO is the only power providing mod present? Probably not. Try it, if it doesn't work feel free to open a feature request issue over on Github.

**Comment by gjgfuj** [comment]
It... Really should. Ender IO uses RF, which OC supports. If it doesn't,
then that's a problem. It works for me.

**Comment by squarism** [comment]
Can you please post the versions of OC and EnderIO you are running?

**Comment by gjgfuj** [comment]
It was fairly old versions of it, but it really should do, and it's a bug
if it doesn't.

**Comment by squarism** [comment]
I rolled back my instance to the older versions

```
OpenComputers-MC1.7.10-1.4.1.14-universal.jar
Mekanism-1.7.10-7.1.1.133.jar
MekanismGenerators-1.7.10-7.1.1.133.jar
MekanismTools-1.7.10-7.1.1.133.jar
```

and it works.  So ... maybe I should change the issue to:

_Current stable / recommended version of Mekanism don't work with stable / recommended version of OC_

**Comment by squarism** [comment]
_update_ Disregard previous "found version that broke it".  My test was flawed.

I've found that whatever MultiMC is doing is out of control.  I don't know the plugin stack it's really using.
- I deleted everything I had in appdata/roaming/.minecraft
- Launched 1.7.10 vanilla to pull the jars (this is a clean install at this point)
- Reinstalled forge to .minecraft
- Created a mods directory.  Put the following versions in there:
  
  Mekanism-1.7.10-7.1.2.159.jar
  MekanismGenerators-1.7.10-7.1.2.159.jar
  MekanismTools-1.7.10-7.1.2.159.jar
  OpenComputers-MC1.7.10-1.4.8.33-universal
- Used the Forge profile from the launcher, let Forget install to .minecraft, previously I would create a new directory.

At this point, I can swap versions of OC and Mekanism around and everything works.  A computer is powered by a cube.  It is WEIRD.  I would understand boot errors.  I would understand this throwing forge errors.  This was crazy track-down-party-time.

I'm closing this as MultiMC's fault.  Thanks for replies, sorry for wasting your time.
